src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStoreV1.swift

```diff
@@ -24,7 +24,7 @@ class ContactDiaryStoreV1: DiaryStoring, DiaryProviding {
 
 	// MARK: - Init
 
-	init(
+	init?(
```

Comment:
if init is only used by make method, why isn't it private?

Reply:
Its used by testing.